src/System.Collections/src/System/Collections/Generic/SortedList.cs

```diff
@@ -453,7 +453,7 @@ public bool ContainsValue(TValue value)
 
             if (arrayIndex < 0 || arrayIndex > array.Length)
             {
-                throw new ArgumentOutOfRangeException("arrayIndex", SR.ArgumentOutOfRange_NeedNonNegNum);
+                throw new ArgumentOutOfRangeException("index", arrayIndex, SR.ArgumentOutOfRange_Index);
```

Comment:
I think you misunderstood me... the ```ICollection<T>.CopyTo``` argument was fine named ```arrayIndex```, but you were then throwing an exception with the string ```"index"``` not ```"arrayIndex"```, so it didn't actually match.